README.md

```diff
@@ -1 +1,85 @@
-# Capstone-project---Conference-page
\ No newline at end of file
+# Cpastone Project for module-1
+
+> This is a complete version of my Capstone project.
+
+My goal here is to Build a personal portfolio site from scratch, focusing on building the mobile version first. take a look at what i'v built [Live Demo](https://l2oukmane.github.io/Capstone-project-Conference-page/). 🙂
+
+## Built With
+
+- Lighthouse (An open-source, automated tool for improving the quality of web pages. It has audits for performance, accessibility, progressive web apps, SEO and more).
+- Webhint (A customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors).
+- Stylelint (A mighty, modern linter that helps you avoid errors and enforce conventions in your styles).
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+The basic requirements for building the executable are:
+
+- A working browser application (Google chrome, Mozilla Fire fox, Microsoft edge ...)
+- VSCode or any other equivalent code editor
+- Node Package Manager (For installing packages like Lighthous, webhint & stylelint used for checking for debugging bad codes before deployment)
+
+# Getting Started
+
```

Comment:
- [ ] Kindly make sure your page is deployed and the live link is added to your README👍